mod_regression/forms.py

```diff
@@ -21,6 +21,7 @@ class CommonTestForm(FlaskForm):
 
     sample_id = SelectField("Sample", coerce=int)
     command = StringField("Command")
+    description = TextAreaField("Description")
```

Comment:
Can we put the length restriction of 1024 on this TextAreaField as well?

Reply:
Sure sure